### DISCUSSION TITLE
large batch upload

### DISCUSSION THREAD

**Discussion by gigisung0503** [FlowmapBlue/FlowmapBlue#89] [Q&A]
Hi, is there a way to upload a large batch, say 900,000 rows of flows not using google spread sheet?

**Comment by ilyabo** [comment]
You can host the data as CSV files on any website with CORS enabled as [described here](https://github.com/FlowmapBlue/FlowmapBlue/issues/18#issuecomment-610063627).
Also, I am working on a new (commercial) tool which will support larger file uploads.  Stay tuned. Or contact me at `ilya AT boyandin.me`  if you want to try it out.

**Comment by ilyabo** [comment]
The new tool supporting larger datasets is available at https://www.flowmap.city/
